deltachat-ffi/src/lib.rs

```diff
@@ -622,6 +623,8 @@ pub unsafe extern "C" fn dc_set_chat_name(
     name: *mut libc::c_char,
 ) -> libc::c_int {
     assert!(!context.is_null());
+    assert!(!name.is_null());
+    assert!(chat_id > 9);
```

Comment:
I am stil not sure what this magic constant 9 is. Maybe define it in constants module?

Reply:
I got this explained to me sometime too: you can see the magic value in deltachat.h, search for DC_CHAT_ID_LAST_SPECIAL.  It's apparently a common pattern to treat IDs up to 9 special.

Reply:
Which is a fair point, this also exists in constants.rs so maybe should be replaced with the name.

Reply:
Did it.